packages/react-sdk/src/core/components/CallLayout/SpeakerLayout.tsx

```diff
@@ -158,6 +153,74 @@ export const SpeakerLayout = ({
   );
 };
 
+type ScrollButtonsProps<T extends HTMLElement> = {
+  scrollWrapper: T | null;
+};
+
+const HorizontalScrollButtons = <T extends HTMLElement>({
+  scrollWrapper,
+}: ScrollButtonsProps<T>) => {
+  const scrollPosition = useHorizontalScrollPosition(scrollWrapper);
+
+  const scrollStartClickHandler = () => {
+    scrollWrapper?.scrollBy({ left: -150, behavior: 'smooth' });
```

Comment:
What is the value of 150px based on? Does it depend on the height of the video element?

Reply:
Not really, just a try-and-error value that looked good-ish when you clicked the button. This behaviour was implemented to indicate that the list is scrollable. Better implementation would be with the usage of `scrollIntoView` API - I might tackle that in the next PR though.